src/hooks/useOrders.ts

```diff
@@ -64,7 +64,8 @@ export function useOrders(): Result {
     const fetchOrders = async (offset: number): Promise<void> => {
       // isLoading is the important one
       // controls ongoing fetching chain
-      if (!userAddress || !networkId || !isLoading) {
+      if (!userAddress || !networkId /*|| !isLoading*/) {
+        // TODO: Do not merge with this, just for doing a quick test!!
```

Comment:
Should this have been pushed?

Reply:
was this @anxolin doing the pending stuff?

Reply:
Ups, this shouldn't go there

Reply:
For the merge to develop (this PR), I think it's fine. It'll be eventually merged with the hotfix to develop too